account_asset_management/models/account_asset.py

```diff
@@ -1064,3 +1064,68 @@ def _compute_entries(self, date_end, check_triggers=False):
                 triggers.sudo().write(recompute_vals)
 
         return (result, error_log)
+
+    @api.model
+    def _xls_acquisition_fields(self):
+        """
+        Update list in custom module to add/drop columns or change order
+        """
+        return [
+            'account', 'name', 'code', 'date_start', 'depreciation_base',
+            'salvage_value',
+        ]
+
+    @api.model
+    def _xls_active_fields(self):
+        """
+        Update list in custom module to add/drop columns or change order
+        """
+        return [
+            'account', 'name', 'code', 'date_start',
+            'depreciation_base', 'salvage_value',
+            'period_start_value', 'period_depr', 'period_end_value',
+            'period_end_depr',
+            'method', 'method_number', 'prorata', 'state',
+        ]
+
+    @api.model
+    def _xls_removal_fields(self):
+        """
+        Update list in custom module to add/drop columns or change order
+        """
+        return [
+            'account', 'name', 'code', 'date_remove', 'depreciation_base',
+            'salvage_value',
+        ]
+
+    @api.model
+    def _xls_asset_template(self):
```

Comment:
@luc-demeyer out of curiosity, why did you design these methods to only return a dictionary with which you update the template instead of passing the template to these methods and using the result that it returns?

```
@api.model
def _xls_asset_template(self, template):
    """ Allow overrides to manipulate the template for assets """
    return template
```

It would seem to me that that would allow for more flexibility.